### ISSUE TITLE
How to set FLAC compression level?

### ISSUE THREAD

**Issue by elendil7** [yt-dlp/yt-dlp#8988]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

Is it possible to set the compression level when using the --audio-quality flag in conjunction with the --extract-audio and --audio-format flag?

For example, would the following convert the extracted audio to FLAC with compression level of 8?
```--extract-audio --audio-format flac --audio-quality 8```

Thanks

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

**Comment by pukkandan** [comment]
No, currently it doesn't. What'd be the equivalent ffmpeg command to what you want to achieve?

**Comment by elendil7** [comment]
Example:
```ffmpeg -i input.wav -c:a flac -compression_level 6 output.flac```

input.wav can be any format - whichever audio of the highest quality that ytdlp fetches.

**Comment by pukkandan** [comment]
Note to self: https://ffmpeg.org/ffmpeg-all.html#flac-2

**Comment by yaynstuff** [comment]
Last touched at end of 2024? End of 2025 is closing... Wanted to know the answer to this one too... Where are things at?

**Comment by bashonly** [comment]
> Where are things at?

PR welcome
